config.lua

```diff
@@ -0,0 +1,3 @@
+Config = Config or {}
```

Comment:
> Is this PR finished?
> 
> You should probably apply a proper title and PR template so that I can see the goal of this PR.

For now it's just a concept, I want to get feedback if everyone likes that idea.

Reply:
> For now it's just a concept, I want to get feedback if everyone likes that idea.

Yes, but what is that concept?

Reply:
I assume the concept being changing from the Skyrim like lock pick mini game to the ox_lib circle mini game? The Skyrim one seems more interesting to me, but the ox_lib is of course simpler. I think I'd lean towards keeping the Skyrim one.

Reply:
I assume that's the context as well, what is so WIP about it in this case, it looks pretty much done if it is only ox_lib support.

I'm leaning more towards the original lockpicking mission, however, I don't really care either way.